src/main/kotlin/com/wire/apps/polls/services/ProxySenderService.kt

```diff
@@ -0,0 +1,35 @@
+package com.wire.apps.polls.services
+
+import com.wire.integrations.jvm.exception.WireException
+import com.wire.integrations.jvm.model.QualifiedId
+import com.wire.integrations.jvm.model.WireMessage
+import com.wire.integrations.jvm.service.WireApplicationManager
+import mu.KLogging
+import pw.forst.katlib.createJson
+
+/**
+ * Service responsible for sending requests to the proxy service Roman.
+ */
+class ProxySenderService {
+    private companion object : KLogging()
+
+    /**
+     * Send given message
+     */
+    suspend fun send(
+        manager: WireApplicationManager,
+        message: WireMessage,
+        conversationId: QualifiedId?
+    ) {
+        logger.debug { "Sending: ${createJson(message)}" }
+        try {
+            conversationId ?: throw NullPointerException("Missing conversation ID.")
```

Comment:
If you accept a nullable valuable but the immediately throwing a NPE (NullPointerException), then it means you are not accepting nullable values. Might as well change conversationId to not-nullable.
Otherwise other services might think they might skip that value and simply put null